### PR TITLE
Rename "Core" to "Basic" changefeeds

### PR DESCRIPTION
**Pull request by rmloveland**
Addresses DOC-10665

Summary of changes:

- A simple search and replace of `s/core changefeed/basic changefeed/g`

- NB. This will also need to be backported in a separate PR to v23.1, v23.2, v24.1, v24.2 docs to coincide with the v24.3 release and associated relicensing

### CODE REVIEW

**Comment by rmloveland** [comment]
@rohan-joshi and @jaiayu IMO y'all are optional on actually reviewing this but I wanted to keep you informed of the change 

it's a pretty straightforward find and replace

@kathancox pls take a look as the docs maintainer of this stuff and make sure I didn't mess up anything with my mostly blind find and replace :-)

File: src/current/v24.3/change-data-capture-overview.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ CockroachDB has two implementations of changefeeds:
 <table class="comparison-chart">
   <tr>
     <th></th>
-    <th>Core changefeeds</th>
+    <th>basic changefeeds</th>
```
Comment (kathancox) [inline src/current/v24.3/change-data-capture-overview.md:29]: This should be capitalized

Comment (kathancox) [inline src/current/v24.3/change-data-capture-overview.md:29]:
```suggestion
    <th>Basic changefeeds</th>
```

File: src/current/v24.3/changefeed-examples.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ toc: true
 docs_area: stream_data
 ---
 
-This page provides step-by-step examples for using Core and {{ site.data.products.enterprise }} changefeeds. Creating {{ site.data.products.enterprise }} changefeeds is available on CockroachDB {{ site.data.products.basic }}, {{ site.data.products.standard }}, {{ site.data.products.advanced }}, and with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) on CockroachDB {{ site.data.products.core }} clusters. Core changefeeds are available in all products.
+This page provides step-by-step examples for using Core and {{ site.data.products.enterprise }} changefeeds. Creating {{ site.data.products.enterprise }} changefeeds is available on CockroachDB {{ site.data.products.basic }}, {{ site.data.products.standard }}, {{ site.data.products.advanced }}, and with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) on CockroachDB {{ site.data.products.core }} clusters. basic changefeeds are available in all products.
```
Comment (kathancox) [inline src/current/v24.3/changefeed-examples.md:8]:
```suggestion
This page provides step-by-step examples for using Core and {{ site.data.products.enterprise }} changefeeds. Creating {{ site.data.products.enterprise }} changefeeds is available on CockroachDB {{ site.data.products.basic }}, {{ site.data.products.standard }}, {{ site.data.products.advanced }}, and with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) on CockroachDB {{ site.data.products.core }} clusters. Basic changefeeds are available in all products.
```

File: src/current/v24.3/changefeed-examples.md [hunk 3]
```diff
@@ -775,17 +775,17 @@ In this example, you'll set up a changefeed for a single-node cluster that is co
 
 <section class="filter-content" markdown="1" data-scope="core">
 
-Core changefeeds stream row-level changes to a client until the underlying SQL connection is closed.
+basic changefeeds stream row-level changes to a client until the underlying SQL connection is closed.
```
Comment (kathancox) [inline src/current/v24.3/changefeed-examples.md:778]:
```suggestion
Basic changefeeds stream row-level changes to a client until the underlying SQL connection is closed.
```

**Review by kathancox — COMMENTED** [review]
I'll wait for @rohan-joshi to confirm this is ok for now.

My opinion is that we're approaching the release very quickly, and so this should work as a stop gap. 

@rohan-joshi We should work on a plan to figure out how the team wants to name both changefeed times and then create an issue for me to work on this perhaps in H1. I think we should look at the `EXPERIMENTAL CHANGEFEED FOR` syntax as a part of this later work.

File: src/current/v24.3/changefeed-examples.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v24.3/changefeed-examples.md:8]: Fixed! ✅

File: src/current/v24.3/changefeed-examples.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v24.3/changefeed-examples.md:778]: ✅

File: src/current/v24.3/change-data-capture-overview.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v24.3/change-data-capture-overview.md:29]: ✅

Comment (rmloveland) [reply src/current/v24.3/change-data-capture-overview.md:29]: Fixed it! thanks

**Comment by rmloveland** [comment]
> I'll wait for @rohan-joshi to confirm this is ok for now.

FWIW Rohan confirmed in a DM with me and Ayushi, i can't share a link but i noted it in [this Jira comment](https://cockroachlabs.atlassian.net/browse/DOC-10665?focusedCommentId=253187)

> My opinion is that we're approaching the release very quickly, and so this should work as a stop gap.

🚀 🙏 

Thanks for the review Kathryn!  I believe i've fixed all the capitalization errors in the latest commit named `Update with kathancox feedback (1)`

**Comment by rohan-joshi** [comment]
lgtm, thanks all

**Pull request merged**